draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -249,6 +249,8 @@ information.
 * `vct`
     * REQUIRED. The type of the Verifiable Credential, e.g.,
 `https://credentials.example.com/identity_credential`, as defined in (#type-claim).
+* `vct#integrity`
+    * OPTIONAL hash of the Type Metadata document to guarantee integrity as defined in Section (#document-integrity). MUST NOT be set if the `vct` claim is not set.
```

Comment:
`vct` is REQUIRED in this context per line 250 above so `if the `vct` claim is not set` is never the case. 
```suggestion
    * OPTIONAL hash of the Type Metadata document to guarantee integrity as defined in Section (#document-integrity).
```

Reply:
It's also required when it's not a URL, but a URI that got fetched via e.g. a database, correct?